compiler/rustc_target/src/spec/targets/thumbv7em_m4f_none_eabihf.rs

```diff
@@ -0,0 +1,10 @@
+use crate::spec::{Target, TargetMetadata};
+
+pub(crate) fn target() -> Target {
+    let mut target = super::thumbv7em_none_eabihf::target();
```

Comment:
@jonathanpallant could you help me figure out what specific options we might want to set here to optimize for the M4F CPU?

Reply:
For the Cortex-M4F, which only has one kind of FPU, you just need target-cpu=cortex-m4.